tests/PageObjects/SongListPage.ts

```diff
@@ -12,6 +12,14 @@ export class SongListPagePO {
     return this.page.getByTestId(`song-${songID}`);
   }
 
+  public async expectSongToBeHidden(songID: string) {
```

Comment:
since this is a simple assertion, you don't have to put it in a page object and do it directly in the test
```
  await expect(pages.songListPage.getSongElement(songID)).toBeVisible();
```

you already do it in restart-song test